dotCMS/src/main/java/com/dotmarketing/viewtools/DotTemplateTool.java

```diff
@@ -1,12 +1,17 @@
 package com.dotmarketing.viewtools;
 
+import java.io.IOException;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.TimeUnit;
 
 import javax.servlet.http.HttpServletRequest;
 
+import com.dotcms.contenttype.transform.JsonTransformer;
+import com.dotcms.repackage.com.fasterxml.jackson.core.JsonParseException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotcms.repackage.com.fasterxml.jackson.core.JsonParseException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)